src/main/java/net/mcreator/ui/component/JMinMaxSpinner.java

```diff
@@ -38,8 +39,8 @@ public JMinMaxSpinner(double minVal, double maxVal, double smin, double smax, do
 		init();
 
 		if (step < 0.001) {
-			((JSpinner.NumberEditor) min.getEditor()).getFormat().setMaximumFractionDigits(4);
-			((JSpinner.NumberEditor) max.getEditor()).getFormat().setMaximumFractionDigits(4);
+			((JSpinner.NumberEditor) min.getEditor()).getFormat().setMaximumFractionDigits(6);

```

Comment:
step < 0.001 should have this at 4, you can add additional if before and change this to else if and check

step < 0.00001 and in this case set to 6

Reply:
✔️ Fixed

Reply:
6,5,6 or 6,5,4?

Use of modulo operator is also less clear than < > for average code reader